etc/adminhtml/system/adyen_testing_performance.xml

```diff
@@ -55,5 +55,27 @@
                 ]]>
             </comment>
         </field>
+        <field id="remove_processed_webhooks" translate="label" type="select" sortOrder="50" showInDefault="1" showInWebsite="0" showInStore="0">
+            <label>Remove processed webhooks</label>
+            <backend_model>Adyen\Payment\Model\Config\Backend\ProcessedWebhookRemoval</backend_model>
+            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+            <config_path>payment/adyen_abstract/remove_processed_webhooks</config_path>
+            <comment model="Adyen\Payment\Model\Comment\WebhookRemovalNotice"/>
+            <tooltip>
+                Processed webhooks older than a configured number of days will be removed from the database by a cronjob if this feature is enabled.
+                The default value is 90 days and this can be configured from the `Number of days to remove processed webhooks` configuration field.
```

Comment:
```suggestion
                The default value is 90 days and this can be configured in the `Number of days to remove processed webhooks` configuration field.
```